README.md

```diff
@@ -63,43 +63,40 @@ For user convenience, this section assumes that users will deploy the infrastruc
-      Attaching to gateway, public-qiskit-serverless-postgres-1, qs-jupyter, ray-head, scheduler
+      Attaching to gateway, public-qiskit-serverless-postgres-1, ray-head, scheduler
       ```
+1. Write your first Qiskit Function following our hello-world example in the [tutorial section](https://qiskit.github.io/qiskit-serverless/getting_started/basic/01_running_program.html).
 
+   That's all!
 
-1. Launch JupyterLab environment.
-   ```shell
-   cd docs/getting_started/ # the directory with sample notebooks
-   jupyter lab
-   ```
-   This will open the Jupyter Lab environment in your web browser.
-1. Write your first example Qiskit Pattern.
-   In the JupyterLab, create a new file, `pattern.py`, in the `work` directory. You can include any arbitrary Python code in your program, or you can use the
-   [example Python file in this tutorial](https://github.com/Qiskit/qiskit-serverless/blob/main/docs/getting_started/basic/01_running_program.ipynb).
-
-1. Run the program
-   In the JupyterLab, create a new notebook in the same directory as your program, and execute [the tutorial code](https://github.com/Qiskit/qiskit-serverless/blob/main/docs/getting_started/basic/01_running_program.ipynb).
-
-   You can check the job status and get the result.
-
-   ```
-   job.status()
-   # 'DONE'
+#### Kubernetes
+For user convenience, this section assumes that users will deploy the infrastructure in a local environment using Kind following the next steps.
 
-   job.logs()
-   # 2023-09-21 03:48:40,286\tINFO worker.py:1329 -- Using address 172.18.0.4:6379 set in the environment variable RAY_ADDRESS\n2023-09-21 03:48:40,286\tINFO worker.py:1458 -- Connecting to existing Ray cluster at address: 172.18.0.4:6379...\n2023-09-21 03:48:40,295\tINFO worker.py:1633 -- Connected to Ray cluster. View the dashboard at \x1b[1m\x1b[32m172.18.0.4:8265 \x1b[39m\x1b[22m\n
-   ```
-   ```
-   job.status()
-   # '{"quasi_dists": [{"1": 0.5071335183298108, "5": 0.4334908044837378, "7": 0.0593756771864515}, {"1": 0.9161860602334094, "5": 0.0838139397665906}, {"2": 0.4999999999999999, "3": 0.4999999999999999}]}'
-   ```
+1. Prepare local Qiskit Serverless infrastructure
+   1. Install Kind
+      To simplify the process to deploy a k8s cluster locally we use [Kind](https://kind.sigs.k8s.io/docs/user/quick-start#installation) as the main tool to create a cluster.
+   1. Install qiskit-serverless on your local system (we recommend using a [virtual environment](https://docs.python.org/3/library/venv.html)).
+      ```shell
+      pip install qiskit-serverless
+      ```
+   1. Clone the Qiskit Serverless repository
+      ```shell
+      git clone https://github.com/Qiskit/qiskit-serverless.git
+      ```
+   1. Run Qiskit Serverless infrastructure
+      Execute the script to setup the cluster
+      ```shell
+      ./docs/deployment/custom_function/local_cluster/deploy.sh
```

Comment:
I tried to change to the `local_cluster` directory and then execute it and it doesn't work. It should be executed from the root.

We can add a `tox` command or something similar to simplify it.

Reply:
Modification applied here: [ae4bb54](https://github.com/Qiskit/qiskit-serverless/pull/1548/commits/ae4bb549669d64236af9180fbb65454666160f30)

The command is:
```shell
tox -e cluster-deploy
```

How do you see the command? Another proposal for the naming? Also @paaragon if you have any other proposal 👍